pkg/security/tests/schemas/exit.schema.json

```diff
@@ -0,0 +1,39 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "exic.json",
```

Comment:
```suggestion
    "$id": "exit.json",
```

Reply:
Thanks for catching this! It's been fixed on https://github.com/DataDog/datadog-agent/pull/12885/commits/72ba90c93f0fe11c61cbb7a5c6b7cb2d408ca9c1.